packages/frontpage/lib/schema.ts

```diff
@@ -197,7 +203,7 @@ export const ConsumedOffset = sqliteTable("consumed_offsets", {
 export const OauthAuthRequest = sqliteTable("oauth_auth_requests", {
   state: text("state").notNull().unique(),
   iss: text("iss").notNull(),
-  did: did("did").notNull(),
+  did: didOrEmptyString("did").notNull(),
```

Comment:
This is unfortunate but it's basically just to get around having to do a database migration in sqlite that'll involve dropping the entire table to remove the notNull constraint 😭